src/server.js

```diff
@@ -39,6 +39,17 @@ app.get('/', (req, res) => {
   res.send(renderPage({ locale, pageComponent: 'Welcome', props: { locale } }))
 })
 
+app.get('/T4', (req, res) => {
+  res.send(
+    renderPage({
+      locale,
+      pageComponent: 'T4',
+      title: 'T4',
```

Comment:
Let's make the title `Your Income` so that it matches the <h1> on the page.

Reply:
oups, I thought I did that! Done now